### PR TITLE
Closes #83: setup eslint + prettier for backend

### LINKED ISSUE #83: Setup Linting and Prettier using Airbnb style for backend
Is your feature request related to a problem? Please describe.
We should have consistent formatting and code style/guidelines across the codebase.

Describe the solution you'd like
Setup linting and prettier for the project so that we keep a consistent code style.

Describe alternatives you've considered
There are other code styles guides like Google, but the Airbnb style guide seems to be the most suitable for React.

Additional context
None

### PR DESCRIPTION
**Pull request by kyzh0**
Closes #83 

Setup prettier and eslint configs for the backend using the Airbnb style guide.
Developers should use ESLint and Prettier extensions for VSCode.

### CODE REVIEW

File: telos-backend/package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "test": "jest"
+    "test": "jest",
+    "pretty": "prettier --write \"./**/*.{js,jsx,json}\""
```
Comment (khal745) [inline telos-backend/package.json]: Did we also want a `lint` command to see changes but not make them? Or not have this functionality at all. Also we should add this to backend readme. But `pretty` command looks good to me.

**Review by khal745 — COMMENTED** [review]
LGTM! Perhaps a lint command is useful, but not necessary - we just need to define our workflow. I think there will be some merge conflicts here re: changing package.json commands etc, so might need to be careful about order we merge your 2 PRs in.

Also when I was running this command locally, all the files were being changed - I think due to changing the new lines as LF instead of CRLF (nothing looks as though it changes). Will investigate this.

File: telos-backend/package.json (hunk 1 above)
Comment (kyzh0) [reply telos-backend/package.json]: Good idea, I've just added it now.

**Review by flexzy — APPROVED** [review]
LGTM

**Review by khal745 — APPROVED** [review]
LGTM

**Pull request merged**